armi/reactor/converters/uniformMesh.py

```diff
@@ -555,6 +584,11 @@ def applyStateToOriginal(self):
         .. impl:: Map select parameters from composites on the new mesh to the original mesh.
             :id: I_ARMI_UMC_PARAM_BACKWARD
             :implements: R_ARMI_UMC_PARAM_BACKWARD
+
+            To ensure that the parameters on the original Reactor are from the converter Reactor,
```

Comment:
```suggestion
            To ensure that the parameters on the original Reactor are from the converted Reactor,
```

I could see this going either way, but I think "converted" reactor is better here